main/ui/ui.go

```diff
@@ -30,9 +31,21 @@ func main() {
 
 	config := common.LoadConfig()
 
-	apiInstance := common.NewAPI(config.API)
-	blueflood := blueflood.NewBlueflood(config.Blueflood)
-	backend := backend.NewSequentialMultiBackend(blueflood)
+	// TODO: create profiler
+	profiler := (*inspect.Profiler)(nil)
+
+	apiInstance := inspect.ProfilingAPI{
+		API:      common.NewAPI(config.API),
+		Profiler: profiler,
+	}
+	blueflood := inspect.ProfilingBackend{
+		Profiler: profiler,
+		Backend:  blueflood.NewBlueflood(config.Blueflood),
+	}
+	backend := inspect.ProfilingMultiBackend{
+		MultiBackend: backend.NewSequentialMultiBackend(blueflood),
```

Comment:
we can make the integrator to provide `func (Backend) MultiBackend` instead of `Multibackend` as an input.